render_config.sh

```diff
@@ -18,23 +13,17 @@ fi
 
 DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
 
-SERVICE_ACCOUNT_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
-    vault read -format=json ${FIRECLOUD_ACCOUNT_VAULT_PATH} | \
-    jq .data`
+if type vault > /dev/null; then
+  SERVICE_ACCOUNT_CREDS=$(VAULT_ADDR="https://clotho.broadinstitute.org:8200" VAULT_TOKEN=$VAULT_TOKEN vault read -format=json ${FIRECLOUD_ACCOUNT_VAULT_PATH} | jq .data)
+else
+  SERVICE_ACCOUNT_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
+      vault read -format=json ${FIRECLOUD_ACCOUNT_VAULT_PATH} | \
+      jq .data`
+fi
 
 if [[ -z "$SERVICE_ACCOUNT_CREDS" ]]; then
     printf "\033[0;31m Could not fetch service account creds. Check your vault token. \n\033[0m"
     exit 1
 fi
 
 echo "$SERVICE_ACCOUNT_CREDS" > ${SERVICE_ACCOUNT_OUTPUT_FILE_PATH}
```

Comment:
This line currently fails, since the `rendered` directory doesn't yet exist.
`./render_config.sh: line 29: ./src/test/resources/rendered/wsm-firecloud-account.json: No such file or directory`